10.IDR_metadata/scripts/create_processed.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python
```

Comment:
This looks like a duplicate file that needs to be deleted (there is no number prefix)